src/Listener/SaveTrelloIdToDatabase.php

```diff
@@ -78,4 +80,9 @@ private function prefixContentWithUrl(Discussion $discussion): string
 
         return "[Original post]($url)\n\n".$discussion->posts->first()->content;
     }
+
+    private function rememberLastUsedLaneId(string $lane): void
+    {
+        $this->settings->set('blomstra-trello.last_used_lane_id', $lane);
```

Comment:
Rather than setting this each time, perhaps get the current value of the key and only set if it's different?